.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-json
+      - id: check-merge-conflict
+  - repo: https://github.com/awslabs/git-secrets
+    rev: 80230afa8c8bdeac766a0fece36f95ffaa0be778
+    hooks:
+      - id: git-secrets
+        entry: "git-secrets --scan"
+        files: "."
+  - repo: https://github.com/aws-cloudformation/cfn-python-lint
+    rev: v0.37.0
+    hooks:
+      - id: cfn-python-lint
+        name: AWS CloudFormation Linter
+        files: \.(template)$
+        args: [--ignore-checks=W3002]
```

Comment:
Any checks to ignore should be determined by the project. I'm not convinced that everyone should be ignoring this check.

Reply:
agreed